src/pages/use-cases/service-mesh.jsx

```diff
@@ -48,21 +47,20 @@ const sectionContent4 = {
     'With Cilium Service Mesh, users now have the choice to run a service mesh with sidecars or without them, based on their specific requirements and constraints. This flexibility reduces the complexity and overhead impact of sidecars.',
   videoSrc: 'https://www.youtube.com/embed/WhVyZZ_QkVA',
   whiteBackground: true,
+  imageRight: false,
 };
 
 const ServiceMeshPage = () => (
   <MainLayout>
-    <section className="bg-[#F6F7F8]">
-      <IntroSection {...introContent} />
-      <VideoFeatureSection {...sectionContent1} />
-      <ImageFeatureSection {...sectionContent2} />
-      <VideoFeatureSection {...sectionContent3} />
-      <VideoFeatureSection {...sectionContent4} />
-      <JoinUsCard />
-    </section>
+    <Hero {...heroContent} />
+    <FeatureSection {...sectionContent1} className="pb-16 md:pb-16 lg:pb-16" />
+    <FeatureSection {...sectionContent2} />
+    <FeatureSection {...sectionContent3} className="pb-16 md:pb-16 lg:pb-16" />
+    <FeatureSection {...sectionContent4} />
+    <JoinUsCard />
   </MainLayout>
 );
 
 export default ServiceMeshPage;
 
-export const Head = () => <title>{introContent.title}</title>;
+export const Head = () => <title>{heroContent.title}</title>;
```

Comment:
I believe we have the SEO component for using in Head function, please check it out https://github.com/cilium/cilium.io/blob/main/src/pages/brand.jsx#L25-L28